Zend/zend_compile.c

```diff
@@ -7583,16 +7663,14 @@ static void zend_compile_prop_group(zend_ast *ast) /* {{{ */
 }
 /* }}} */
 
-static void zend_check_const_and_trait_alias_attr(uint32_t attr, const char* entity) /* {{{ */
+static void zend_check_trait_alias_modifiers(uint32_t attr) /* {{{ */
 {
 	if (attr & ZEND_ACC_STATIC) {
-		zend_error_noreturn(E_COMPILE_ERROR, "Cannot use 'static' as %s modifier", entity);
+		zend_error_noreturn(E_COMPILE_ERROR, "Cannot use 'static' as method modifier");
```

Comment:
Nit: would it make sense to further clarify the error message so that it's immediately clear that it is related to traits? Another even smaller nit: during PHP 8.0, I made tons of error messages use `"` instead of `'`, so it might be a somewhat good chance to continue the project.

Reply:
Absolutely!